src/components/Buy/BuySuccessPage.jsx

```diff
@@ -11,41 +19,109 @@ function BuySuccessPage() {
+  ];
+
+  // Fill in the schedule with data from organizationData
+  if (organizationData && organizationData.schedule) {
+    organizationData.schedule.forEach((day, index) => {
+      if (day && day.start && day.end) {
+        completeSchedule[index] = {
+          day: getDayName(index),
+          start: day.start,
+          end: day.end
+        };
+      }
+    });
+  }
+
+  // Filter out days with missing times and format the schedule string
+  const scheduleString = completeSchedule
+  .filter(day => day.start && day.end) // Exclude days with missing times
+  .map(day => `${day.day}: ${day.start} - ${day.end}`)
+  .join('; ');
+
   return (
     <Container fluid className="p-4">
-      <Col md={8} className="mb-4">
+      <Col md={12} className="mb-4 d-flex flex-column align-items-center justify-content-center">
         <Container className="next-steps-container">
           <Row className="justify-content-center">
-            <Col md={12} className="mb-4">
+            <Col md={10} className="mb-4 d-flex flex-column align-items-center justify-content-center">
               <Card className="next-steps-card">
                 <Card.Header as="h5">You will be contacted to set up an exchange!</Card.Header>
                 <Card.Body>
                   <Card.Text>
-                    The Seller has been notified that you are interested in their textbook! They will accept or decline
-                    the request to purchase this textbook and you will be notified. If they accept, we will provide them
-                    your contact information and they will reach out to you to organize an exchange. This should occur
-                    within the next week.
-                  </Card.Text>
-                  <Card.Text>
-                    Exchanges must occur at [location of exchange] between the hours of [time of exchange]. Please
-                    confirm on the payment method used in advance, and make the payment in person during the exchange.
-                    If doing an electronic payment, you must exchange handles with the seller.
+                    <p>
+                      The Seller has been notified that you are interested in their textbook! They will accept or decline
+                      the request to purchase this textbook and you will be notified. If they accept, we will provide them
+                      your contact information and they will reach out to you to organize an exchange. This should occur
+                      within the next week.
+                    </p>
+                    <p>
+                      Exchanges must occur at {organizationData?.exchange_location || 'Location not available'} between the hours of {scheduleString}. Please
+                      confirm on the payment method used in advance, and make the payment in person during the exchange.
+                      If doing an electronic payment, you must exchange handles with the seller.
+                    </p>
```

Comment:
I would keep a card.text here instead of a <p> tag, I got eslint errors from using a paragraph tag here. And to just be consistent